README.md

```diff
@@ -1,43 +1,39 @@
 [comment]: # ( Copyright Contributors to the Open Cluster Management project )
 
 # Governance Policy Propagator [![KinD tests](https://github.com/open-cluster-management/governance-policy-propagator/actions/workflows/kind.yml/badge.svg?branch=main&event=push)](https://github.com/open-cluster-management/governance-policy-propagator/actions/workflows/kind.yml)
-Red Hat Advanced Cluster Management Governance - Policy Propagator
 
-## How it works
+## Description
 
-This operator watches for following changes to trigger reconcile
+The governance policy propagator is a controller that watches `Policies`, `PlacementBindings`, and `PlacementRules`. It manages replicated Policies in cluster namespaces based on the PlacementBindings and PlacementRules, and it updates the status on Policies to show aggregated cluster compliance results. The controller can be run as a stand-alone program or as an integrated part of governing risk with the Open Cluster Management project.
```

Comment:
`The controller can be run as a stand-alone program or as an integrated part of governing risk with the Open Cluster Management project.`
I don't think this is appropriate for this repo. I think it should be:

This controller is a part of [governance-policy-framework](https://github.com/open-cluster-management/governance-policy-framework)